### PR TITLE
Issue #39:  support Find on alternate labels; surface alternate labels in tooltip.

### PR DESCRIPTION
**Pull request by kodecharlie**
We need to test more before resolving this WIP PR.  NOTE:  despite the commit-text below, this is for issue-#39.

### CODE REVIEW

**Comment by aclarkxyz** [comment]
The first thing I did for testing was to try "Homo sap" and "human", and I got a lot more matches than I wanted...

... I think we need to take out searching of the description...  I probably included that originally because we didn't have altLabels, so that was sort of a proxy. Or maybe the lookup panel can have an extra checkbox for "[x] Match descriptions" which defaults to off.

File: src/com/cdd/bao/util/URIRowLine.java [hunk 1]
```diff
@@ -55,6 +59,23 @@ public URIRowLine(String uri, String tooltip, int focusIndex, int padding, URIRo
 		Button lookupUriBtn = new Button("Lookup");
 		lookupUriBtn.setOnAction(event -> this.delegate.actionLookupName(focusIndex));
 
+		// tooltip: abbreviated uri + alternate labels
```
Comment (aclarkxyz) [inline src/com/cdd/bao/util/URIRowLine.java]: Don't need a tooltip for the button

**Comment by aclarkxyz** [comment]
Don't need a tooltip on the "Lookup" button. The action is in the LookupPanel: we need some way for the user to see what the altLabels are. I was originally thinking that having a tooltip for each displayed row would be good enough. It might also be a good idea to add another column for "Other Labels", and add in any content (comma separated should be fine).

**Comment by aclarkxyz** [comment]
The subject area for this issue isn't the "Search" feature (Cmd-F), it's the LookupPanel, i.e. this thing:

image

... so it may be better to start with a new branch, because adding this functionality to the search dialog is counterproductive. The idea is to have the LookupPanel search labels+altLabels (and *not* search the description (which it does currently), unless you want to add a checkbox for that). Adding altLabel to the search criteria means that the user should have some way to see why the match happened, hence having a tooltip on the line items in the dialog might help. Adding an additional column to those items is also an option.

**Comment by kodecharlie** [comment]
So, I don't have a strong opinion on whether Cmd-F dialog shows the "Include descriptions in search" checkbox or not.  It's cool, but I'm not clear on whether it has value for end-users.

Sure, I will open a separate branch to knock out the alt-labels for Lookup dialogs.

**Pull request merged**